src/components/ebay-textbox/README.md

```diff
@@ -26,3 +26,4 @@ Event | Data | Description
 `textbox-input` | `{ originalEvent, value }` |
 `textbox-focus` | `{ originalEvent, value }` |
 `textbox-blur` | `{ originalEvent, value }` |
+`textbox-floating-label-init` | `{ null, value }` |
```

Comment:
It feels a little odd, as an end user, to receive a null value as first param.

Reply:
It would actually (by the looks of it) be an object with `{ originalEvent: null, value: null }`. I think the docs are wrong here.

Reply:
Aha! Thanks Dylan. Yes, let's fix those docs.